### PR TITLE
Clarification on trying to wield items one-handed

### PR DESCRIPTION
**Pull request by Night-Pryanik**
Separate messages for wielding items that are handled only two-handed and if an item is too heavy for wielding it one-handed.

### CODE REVIEW

File: src/player.cpp [hunk 1]
```diff
@@ -10239,11 +10239,18 @@ bool player::wield(item* it, bool autodrop)
...
     }
 
-    if( it->is_two_handed(*this) && !has_two_arms() ) {
-        add_msg(m_info, _("You cannot wield a %s with only one arm."),
+    if (!has_two_arms()) {
+        if (it->has_flag("ALWAYS_TWOHAND")) {
+        add_msg(m_info, _("You cannot wield a %s with only one arm, as it is handled only two-handed."),
                 it->tname().c_str());
         return false;
+        } else if (it->weight() / 113 > str_cur * 4) {
```
Comment (Coolthulhu) [inline src/player.cpp]: This strength check should be in a function. Copy+pasted formulas are easy to miss when updating something and this could lead to discrepancies.
It would be better like this:
```
if( it->is_two_handed(*this) && !has_two_arms() ) {
    if (it->has_flag("ALWAYS_TWOHAND")) {
        two handed msg here
    } else {
        too heavy message here
    }
```

File: src/player.cpp [hunk 2]
```diff
@@ -10239,11 +10239,18 @@ bool player::wield(item* it, bool autodrop)
...
+        add_msg(m_info, _("You cannot wield a %s with only one arm, as your strength is insufficient to wield it one-handed."),
```
Comment (Coolthulhu) [inline src/player.cpp]: The message is a bit too "formal"
I'd rephrase it to
"You are too wield %s with only one arm"

File: src/player.cpp [hunk 3]
```diff
@@ -10239,11 +10239,18 @@ bool player::wield(item* it, bool autodrop)
         return false;
     }
 
-    if( it->is_two_handed(*this) && !has_two_arms() ) {
-        add_msg(m_info, _("You cannot wield a %s with only one arm."),
+    if (!has_two_arms()) {
+        if (it->has_flag("ALWAYS_TWOHAND")) {
+        add_msg(m_info, _("You cannot wield a %s with only one arm, as it is handled only two-handed."),
```
Comment (Coolthulhu) [inline src/player.cpp]: Similar here. Should be simpler, like
"The %s can't be wielded with only one arm"

File: src/player.cpp [hunk 4]
```diff
@@ -10239,11 +10239,16 @@ bool player::wield(item* it, bool autodrop)
         return false;
     }
 
-    if( it->is_two_handed(*this) && !has_two_arms() ) {
-        add_msg(m_info, _("You cannot wield a %s with only one arm."),
+    if ( it->is_two_handed(*this) && !has_two_arms() ) {
+        if (it->has_flag("ALWAYS_TWOHAND")) {
+            add_msg(m_info, _("The %s can't be wielded with only one arm."),
                 it->tname().c_str());
-        return false;
```
Comment (BevapDin) [inline src/player.cpp]: Why is this `return` statement gone? I think it was important. It looks as if the function now allows you to wield the item anyway, in contrast to the messages.

**Comment by Night-Pryanik** [comment]
Well, it looks like my fault.
